components/layout/flow_list.rs

```diff
@@ -30,15 +30,16 @@ impl Serialize for FlowList {
             let flow_val = ObjectBuilder::new()
                 .insert("class", f.class())
                 .insert("data", match f.class() {
-                    FlowClass::Block => to_value(f.as_block()),
-                    FlowClass::Inline => to_value(f.as_inline()),
-                    FlowClass::Table => to_value(f.as_table()),
-                    FlowClass::TableWrapper => to_value(f.as_table_wrapper()),
+                    FlowClass::Block         => to_value(f.as_block()),
```

Comment:
I'm not sure what the preferred formatting here is - but probably best to leave it as it was.